pkg/test/util/supportedversion/supportedversion.go

```diff
@@ -55,13 +63,32 @@ func init() {
 	New = List[0].Name
 }
 
+func parseVersion(version string) (int, int, int) {
+	// The version can have this formats: "1.22.2", "1.23.0-rc.1", "1.24-alpha"
+	re := regexp.MustCompile(`^(\d+)\.(\d+)\.?(\d*)`)
+
+	matches := re.FindStringSubmatch(version)
+	if len(matches) < 4 {
+		return 0, 0, 0
+	}
+
+	major, _ := strconv.Atoi(matches[1])
+	minor, _ := strconv.Atoi(matches[2])
+	patch, _ := strconv.Atoi(matches[3])
+
+	return major, minor, patch
+}
+
 type Versions struct {
 	Versions []VersionInfo `json:"versions"`
 }
 
 type VersionInfo struct {
 	Name    string   `json:"name"`
 	Version string   `json:"version"`
+	Major   int      `json:"major"`
+	Minor   int      `json:"minor"`
+	Patch   int      `json:"patch"`
```

Comment:
@luksa Hi, I added a way to parse the Major, Minor and Patch version to allow skip test later based on istio version being tested